frontend/scripts/index.js

```diff
@@ -25,6 +24,9 @@ if (process.env.NODE_ENV !== 'production' && PERF_TEST) {
 }
 
 if (process.env.NODE_ENV !== 'production' && REDUX_LOGGER_ENABLED) {
+  // eslint-disable-next-line global-require,import/no-extraneous-dependencies
+  const { createLogger } = require('redux-logger');
```

Comment:
I like this, but then lets use also global require for React in the line 21 of the same file.

Reply:
:+1: I agree